.github/workflows/codeflare-sdk-release-sync.yml

```
@@ -0,0 +1,80 @@
+      with:
+        python-version: |
+          3.8
+          3.9
+
+    - name: Install pipenv
+      run: pip install pipenv
+
+    - name: make sync 
+      run: |
+        git config --global user.email "abhijeetdhumal652@gmai.com"
+        git config --global user.name "abhijeet-dhumal" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)